tests/inference/test_chat_template.py

```diff
@@ -0,0 +1,187 @@
+#
+# Copyright 2023 The LLM-on-Ray Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import pathlib
+
+import pytest
+from transformers import AutoTokenizer
+
+from llm_on_ray.inference.utils import parse_jinja_file
+
+default_jinja_path = (
```

Comment:
I don't think you need to duplicate code here, you just need a shared prefix and a name list.